### PR TITLE
[plugin.video.plutotv] 1.0.9

### PR DESCRIPTION
**Pull request by Lunatixz**
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.video.plutotv/resources/lib/plutotv.py [hunk 1]
```diff
@@ -41,23 +40,27 @@
...
+BASE_LINEUP  = BASE_API + '/v1/channels.json'
+BASE_GUIDE   = BASE_API + '/v1/timelines/%s.000Z/%s.000Z/matrix.json'
+LOGIN_URL    = BASE_API + '/v1/auth/local'
+BASE_CLIPS   = BASE_API + '/v2/episodes/%s/clips.json'
 PLUTO_MENU   = [("Browse Channels" , BASE_LINEUP, 0),
                 ("Browse OnDemand" , BASE_LINEUP, 1),
                 ("Channel Guide"   , BASE_LINEUP, 20)]
               
+def isUWP():
+    return len(fnmatch.filter(glob.glob(LANGUAGE(30011)),'*.*') + fnmatch.filter(glob.glob(LANGUAGE(30012)),'*.*')) > 0
```
Comment (afedchin) [inline plugin.video.plutotv/resources/lib/plutotv.py]:
```python
def isUWP():
    sys.platform == "win10"
```

Comment (Lunatixz) [reply plugin.video.plutotv/resources/lib/plutotv.py]: Thanks, however "Win10" doesn't include Xbox UWP. Also not all "Win10" users installed through the app store so they may not be running the UWP version of Kodi.

Comment (afedchin) [reply plugin.video.plutotv/resources/lib/plutotv.py]: `win10` means uwp platform, on PC, Xbox or mobile

Comment (afedchin) [reply plugin.video.plutotv/resources/lib/plutotv.py]: on native windows 10 it has `win32` value

Comment (Lunatixz) [reply plugin.video.plutotv/resources/lib/plutotv.py]: Okay cool... I'll use this method: thanks.

**Pull request merged**